pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.autopep8]
+max_line_length = 100
+# This should match ruff excludes.
+exclude = "*/allspice/parse/**/*_binary_*.py , */allspice/parse/**/*_variant_*.py"
```

Comment:
These files don't exist in this repo. Same below.

Reply:
Oops, I `wget`'d the file and didn't change it. Fixed now.